styles/books/pl-economics/book.scss

```diff
@@ -201,6 +201,7 @@ $bandColor: #217867;
 @include use('FootnoteCall', 'common:::FootnoteCallShape');
 @include use('TocNoUnit', 'cosmos:::TocNoUnitShape');
 @include use('Equation', 'cosmos:::EquationShape');
+@include use('PageWithIndentation', 'cosmos:::ModuleWithIndentationPolishShape');
```

Comment:
If you have any other ideas for names let me know :wink: I'm not really sure about these ones.

Reply:
@wilkus27 Maybe the shape name can be changed a little e.g. `ModuleWithIndentationForPolishBooksShape`?